WordPress/gutenberg
# Mobile BottomSheet: Adding max-width and centering.

## Description
This PR implements https://github.com/wordpress-mobile/gutenberg-mobile/issues/572

![centered-bottom-sheet](https://user-images.githubusercontent.com/9772967/52811342-24f7f100-3095-11e9-9391-1e36d88711c8.gif)
![simulator screen shot - ipad pro 9 7-inch - 2019-02-14 at 16 45 25](https://user-images.githubusercontent.com/9772967/52811366-3214e000-3095-11e9-874e-f8c5628959c9.png)

This PR doesn't contemplate the space between the bottom of the screen and the BottomSheet.
Don't worry about the separators going out of the bottom sheet. That is resolved here: https://github.com/WordPress/gutenberg/pull/13855

To test:
- Checkout the related `gutenberg-mobile` PR (https://github.com/wordpress-mobile/gutenberg-mobile/pull/600)
- Run the project on iOS and on Android.
- Open the Image settings from an image block.
- If you are running on an iPhone on portrait mode, turn it to landscape.
- Check that the bottom sheet is not covering the full width on "wide" screens.
  - Considered wide: Any phone on landscape, any tablet.
